data/lang/equipment-core/en.json

```diff
@@ -287,46 +327,202 @@
+    "message": "Mass produced guided missile which will attempt to home in on a selected target. Depending on the exact model, this missile may use a variety of homing technologies including one or more of Radar, Laser, or LIDAR. Nevertheless, the technology in these missiles is not guaranteed to ensure a hit and most are quite vulnerable to ECM or other countermeasures."
+  },
+
   "MISSILE_NAVAL": {
     "description": "",
     "message": "Naval Missile"
   },
+  "MISSILE_NAVAL_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "Fire-and-forget homing missile highly resistant to all known types of countermeasures."
+  },
+  "MISSILE_NAVAL_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "These missiles are produced in highly specialised and secretive naval factories in tightly controlled facilities and generally not available to non-military personel. Able to resist most known types of countermeasures, these missiles are fast for their size and usually pack exotic warheads for extreme destructive potential. Often specialised equipment is required in the ships carrying these in order to correctly arm and program the missiles prior to launch."
+  },
+
   "MISSILE_RACK_221": {
     "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
     "message": "LH-120 Hydri Missile Rack"
   },
+  "MISSILE_RACK_221_DESCRIPTION": {
+    "description": "Equipment tooltip",
+    "message": "External missile rack with a capacity of 2 small missiles."
+  },
+  "MISSILE_RACK_221_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "This quality missile rack from Hydri allows you to carry more bangs per hardpoint, essential for taking on those pesky pirates."
+  },
+
   "MISSILE_RACK_322": {
     "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
-    "message": "LH-230 Hydri Missile Rack"
+    "message": "LH-220 Hydri Missile Rack"
+  },
+  "MISSILE_RACK_322_DESCRIPTION": {
+    "description": "Equipment tooltip",
+    "message": "External missile rack with a capacity of 2 medium missiles."
+  },
+  "MISSILE_RACK_322_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "A larger missile rack for the discerning Hydri customer. Bigger bangs for your bucks!"
   },
+
   "MISSILE_RACK_341": {
     "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
     "message": "LH-140 Hydri Missile Rack"
   },
+  "MISSILE_RACK_341_DESCRIPTION": {
+    "description": "Equipment tooltip",
+    "message": "External missile rack with a capacity of 4 small missiles."
+  },
+  "MISSILE_RACK_341_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "This one is a real beauty, coming straight to you from the R&D labs at Hydri. We've squeezed even more missiles into this rack to allow you to really make some fireworks!"
```

Comment:
I had originally intended Hydri to be a brand name similar to the 'Tacoma' in Toyota Tacoma (with the actual manufacturer as-yet undefined), but this is fine. I've added Hydri Defense as a manufacturer - feel free to update the descriptions as makes sense.

Reply:
These were only placeholder suggestions; I was tired and got a bit silly..  don't let me influence the lore direction!

That said, it would be kinda nifty if we could have different manufacturers with perhaps slightly different specs for each piece of equipment, with different manufacturers being more or less popular in different regions of space. Might be an awful lot of effort for negligible gain though..

Reply:
... and I just realised Kongsberg is a real defence manufacturer, not an in-game one :P

Reply:
> That said, it would be kinda nifty if we could have different manufacturers with perhaps slightly different specs for each piece of equipment,

I do agree with this, and it's a direction we've at least sketched out on the Manufacturers page of Notion. However, I think it doesn't make a whole lot of sense to go that route for certain specific types of equipment, like these adapters - it's a lot of work to make extra items, and it's very easy to create a "local minima" where one specific piece of equipment / manufacturer is always chosen and the rest are ignored.